### PR TITLE
[ONB-70] Add disable activation link button/model to table

### PR DESCRIPTION
**Pull request by goktug-gurbuzturk-wp**

### CODE REVIEW

**Review by weaverjess — CHANGES_REQUESTED** [review]
Will slack @goktug-gurbuzturk-wp

File: src/components/subscriptions/buttons/DeleteActivationButton.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  } = useContext(SubscriptionDetailContext);
+  const {
+    enterpriseId, errors, setErrors,
+  } = useContext(SubscriptionContext);
+
+  const disableActivationLink = async () => {
+    await LmsApiService.disableActivationLink(enterpriseId, activationLinkId)
+      .then(() => {
+        forceRefreshDetailView();
+        addToast('Activation link is disabled successfully.');
```
Comment (jakub-koza-wp) [inline src/components/subscriptions/buttons/DeleteActivationButton.jsx:32]:
```suggestion
        addToast('Activation link was disabled successfully.');
```

File: src/components/subscriptions/buttons/DeleteActivationButton.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        });
+      });
+  };
+
+  return (
+    <>
+      <Icon role="button" src={Delete} onClick={open} />
+
+      <AlertModal
+        title="Are you sure want to disable!"
```
Comment (jakub-koza-wp) [inline src/components/subscriptions/buttons/DeleteActivationButton.jsx:48]: I would suggest changing the wording a little
```suggestion
        title="Are you sure you want to disable this link?"
```

File: src/components/subscriptions/buttons/DeleteActivationButton.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          <ActionRow>
+            <Button variant="tertiary" onClick={close}>Cancel</Button>
+            <Button variant="danger" onClick={disableActivationLink}>Disable</Button>
+          </ActionRow>
+        )}
+      >
+        <p>
+          If you disable activation link, the user will not be able to activate the account by
+          clicking the activation link in the received email. This action cannot be undone. If you want
+          to continue with the disabling, please click the Disable button.
```
Comment (jakub-koza-wp) [inline src/components/subscriptions/buttons/DeleteActivationButton.jsx:62]: I think we can drop the last sentence. Actions seem obvious to me from just `Cancel`/`Disable` button naming.
```suggestion
          clicking the activation link in the received email. This action cannot be undone.
```

File: src/components/subscriptions/buttons/DeleteActivationButton.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        onClose={close}
+        footerNode={(
+          <ActionRow>
+            <Button variant="tertiary" onClick={close}>Cancel</Button>
+            <Button variant="danger" onClick={disableActivationLink}>Disable</Button>
+          </ActionRow>
+        )}
+      >
+        <p>
+          If you disable activation link, the user will not be able to activate the account by
```
Comment (weaverjess) [inline src/components/subscriptions/buttons/DeleteActivationButton.jsx:60]: "If you disable this activation link, the user will no longer be able activate their account via the activation link they were emailed. This action cannot be undone."

**Review by weaverjess — COMMENTED** [review]
Sorry, one last change on the words then this is good to go.

File: src/components/subscriptions/buttons/DeleteActivationButton.jsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        onClose={close}
+        footerNode={(
+          <ActionRow>
+            <Button variant="tertiary" onClick={close}>Cancel</Button>
+            <Button variant="danger" onClick={disableActivationLink}>Disable</Button>
+          </ActionRow>
+        )}
+      >
+        <p>
+          If you disable this activation link, the user will no longer be able activate
```
Comment (jakub-koza-wp) [inline src/components/subscriptions/buttons/DeleteActivationButton.jsx:60]: I think that `to` should still be there
```suggestion
          If you disable this activation link, the user will no longer be able to activate
```

**Review by weaverjess — APPROVED** [review]

**Pull request merged**